master/internal/api_command.go

```diff
@@ -174,14 +176,27 @@ func (a *apiServer) GetCommands(
 	if err = a.ask(commandsAddr, req, &resp); err != nil {
 		return nil, err
 	}
+	for _, task := range resp.Commands {
```

Comment:
since the command actor is already responsible for converting this to proto, I think we should push these command-related conversions down to `toNotebook`/`toShell`... to try to cut down on the duplication.